server/webapp/WEB-INF/rails.new/app/views/agents/index.html.erb

```diff
@@ -1,8 +1,12 @@
 <%- @view_title = "Agents" -%>
 <div class="page_header" style='overflow:inherit;'>
     <h1 class="entity_title">Agents</h1>
+  <% if @agents_page_feature_toggle %>
+      <a href="/go/admin/agents" class="new_agents_page">Check out the new agents page</a>
+  <% end %>
```

Comment:
The idiomatic way to write this is —

```ruby
<%= link_to(..., class: :new_agents_page) if is_feature_on? %>
```

and make `is_feature_on?` a [helper function](http://api.rubyonrails.org/v4.0.4/classes/AbstractController/Helpers/ClassMethods.html#method-i-helper_method) instead of polluting with instance variables.